src/sentry/tasks/merge.py

```diff
@@ -186,17 +189,13 @@ def merge_groups(
             try:
                 # it's possible to hit an out of range value for counters
                 new_group.update(
-                    times_seen=F("times_seen") + group.times_seen,
+                    times_seen=F("times_seen") + group.times_seen_with_pending,
```

Comment:
Using `new_group.times_seen` here instead of `new_group.times_seen_with_pending` since the buffered updates for the new group will still be active post-merge, and we don't want to be double-counting. Give me a sanity check that this makes sense.